common/lazy-pages/src/lib.rs

```diff
@@ -16,32 +16,30 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-//! Lazy pages support runtime functions
+//! Lazy pages support for runtime.
+
+#![cfg_attr(not(feature = "std"), no_std)]
 
-use crate::Origin;
-use alloc::string::ToString;
 use core::fmt;
+use gear_common::{pages_prefix, Origin};
 use gear_core::{
     ids::ProgramId,
     memory::{HostPointer, Memory, PageNumber, WasmPageNumber},
 };
 use gear_runtime_interface::{gear_ri, RIError};
 use sp_std::vec::Vec;
 
-#[derive(Debug, Clone, PartialEq, Eq, derive_more::Display)]
+extern crate alloc;
+use alloc::string::ToString;
```

Comment:
can't find thing that uses ToString here
Or it's requirement of `derive_more`?

Reply:
https://github.com/gear-tech/gear/blob/2133029010e76c0f63366179d12ab50f87d01f7e/common/lazy-pages/src/lib.rs#L77-L79